src/SemiclassicalOrthogonalPolynomials.jl

```diff
@@ -292,22 +292,33 @@ end
 semijacobi_ldiv(Q::SemiclassicalJacobi, P::SemiclassicalJacobi) = semijacobi_ldiv(Q.t, Q.a, Q.b, Q.c, P)
 function semijacobi_ldiv(Qt, Qa, Qb, Qc, P::SemiclassicalJacobi)
     @assert Qt ≈ P.t
-    (Qt ≈ P.t) && (Qa ≈ P.a) && (Qb ≈ P.b) && (Qc ≈ P.c) && return SymTridiagonal(Ones(∞),Zeros(∞))
-    Δa = Qa-P.a
-    Δb = Qb-P.b
-    Δc = Qc-P.c
+    (Qt ≈ P.t) && (Qa ≈ P.a) && (Qb ≈ P.b) && (Qc ≈ P.c) && return SymTridiagonal(Ones(∞), Zeros(∞))
+    Δa = Qa - P.a
+    Δb = Qb - P.b
+    Δc = Qc - P.c
     if isinteger(Δa) && isinteger(Δb) && isinteger(Δc) # (Δa,Δb,Δc) are integers -> use QR/Cholesky iteratively
-        if ((isone(Δa)||isone(Δa/2)) && iszero(Δb) && iszero(Δc)) || (iszero(Δa) && (isone(Δb)||isone(Δb/2)) && iszero(Δc))  || (iszero(Δa) && iszero(Δb) && (isone(Δc)||isone(Δc/2)))
+        if Δa < 0
+            R = ApplyArray(inv, semijacobi_ldiv(P.t, P.a, P.b, P.c, SemiclassicalJacobi(P.t, Qa, P.b, P.c)))
+            M = ApplyArray(*, semijacobi_ldiv(Qt, Qa, Qb, Qc, SemiclassicalJacobi(P.t, Qa, P.b, P.c)), R)
```

Comment:
Why the need for `M = `?

Reply:
Not sure what you mean. Those `M = ` were already there I think. I split up `M` and `R` into two lines to make it easier to read and make it easier to see that the `ldiv` is being applied in steps (decrementing one parameter, then onto the next).

Reply:
But isn't the `M = ` line just returned? So the variable is never used

Reply:
I'll remove it